packages/preset-common/src/utils/generic-substitutions.ts

```diff
@@ -0,0 +1,10 @@
+import { names } from '@nx/devkit'
+
+export function genericSubstitutions({ name, npmScope }: { name: string; npmScope: string }) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I think this comment line is not needed as we don't `require()` here

Reply:
nice catch. removed